doc/dev.md

```diff
@@ -24,68 +24,50 @@ Pull request
 Push
 ```
 
-### Running the GitHub pipeline locally
+## Running the GitHub pipeline locally
 
-You will need the following:
+In `docker-compose.yml` change the following:
 
-1. The GitHub App ID of the app you created
-2. The `pem` file containing the private key associated to the app
-3. A comma separated list of GitHub accounts to allow - this could start out as just your GitHub account
-4. A capability file for submitting jobs to a cluster, in this case the main ocaml-ci cluster as documented in https://github.com/ocurrent/ocluster#admin
-5. The app webhook secret that is used to authenticate to the app
+1. Under `services`, then `service`, change the argument to `--github-app-id` to the GitHub App ID of the app you created
+2. Under the same `service` tag, change the argument to `--github-account-allowlist` to a comma-separated list of GitHub accounts to allow—this could start out as just your GitHub account
+3. Under `secrets`, set `ocaml-ci-github-key` to the `pem` file containing the private key associated to the app, and `ocaml-ci-webhook-secret` to a file containing the webhook secret that is used to authenticate to the app
 
+Build the docker containers for `service` and `web`:
 ```
-dune exec -- ocaml-ci-service \
-  --github-app-id <your-github-app-id> \
-  --github-private-key-file <path-to/private-key.pem> \
-  --github-account-allowlist <your-github-username> \
-  --submission-service <path-to-the-submission-capability-file> \
-  --github-webhook-secret-file <path-to-the-app-secret> \
-  --capnp-listen-address tcp:127.0.0.1:9001
-  --migration-path "$PWD/migrations"
+docker build -t ocaml-ci-service -f Dockerfile .
```

Comment:
Is there a reason to have these build steps as separate ones, rather than doing it within the docker-compose file automatically, using something like the following?

```
    build:
      context: .
      dockerfile: Dockerfile   # Dockerfile.web
```